### ISSUE TITLE
--continue problem

### ISSUE THREAD

**Issue by HalitEmir00** [brichard19/BitCrack#297]
How exactly can i get it to continue where i left off when i restart? I've tried to do the --continue but i can't say i was successful. Can someone help me to do this?

**Comment by marssystems** [comment]
You need to do --continue followed by a filename  where the data will be saved -

--continue save.txt

You will need to cerate the save.txt file first.

**Comment by HalitEmir00** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/66889138/106385249-55e7a280-63e0-11eb-8b51-52e48ca13dd5.png)

At first i tried that way, but after it said 'loading 'FİLE'' it stopped progress every time.

I also tried to changing the name of the file. I think there is a problem with finding the file.

**Comment by marssystems** [comment]
Try putting the --continue start1.txt - before -   -i  btc2.txt and see how that goes. Works foe me!

**Comment by HalitEmir00** [comment]
![Screenshot_2](https://user-images.githubusercontent.com/66889138/106386196-df00d880-63e4-11eb-9191-2ec946eacc8f.png)

Stil says the same thing.

I'm using Notepad++. Could there be a problem caused by this?

**Comment by marssystems** [comment]
No - I don't think so. I use notepad++ as well, with no problems.

It might not like the txt extension. Rename the start1.txt to start.1 and see how that goes.

**Comment by HalitEmir00** [comment]
![Screenshot_4](https://user-images.githubusercontent.com/66889138/106386547-8a5e5d00-63e6-11eb-94b1-1fd4c9cf883d.png)

Nothing has changed.

Do i need to write something in the file?

**Comment by marssystems** [comment]
No - it should be empty.
The program will write checkpoint data to it every 10 mins or so,

**Comment by HalitEmir00** [comment]
Any other idea?

**Comment by marssystems** [comment]
Sorry - no. I've not had a problem with this - it just works for me without any issues.

Are you running this as a .bat or are you typing the code into a CMD window directly?

**Comment by HalitEmir00** [comment]
First of all, I find the bitcrack file from CMD. Then i am typing the code and run it.

**Comment by marssystems** [comment]
OK - in your Bitcrack-master folder, create a batch file called crack.bat and put your code in there.
After your code, on the next line put pause then save the file and run it.

Are you using a GPU for this or just your CPU?

**Comment by HalitEmir00** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/66889138/106387247-ebd3fb00-63e9-11eb-97b9-b6475a9b6aa6.png)

It says that and closes itself.

I am using RTX 2070 for this.

**Comment by marssystems** [comment]
OK - well I don't know why its not working for you.
Maybe someone else can help you. Good luck!

**Comment by MisterTeo** [comment]
Try to compile in Release mode latest version https://github.com/brichard19/BitCrack/releases maybe is "master" release the problem....

**Comment by SirDenis** [comment]
C:\123>clBitCrack.exe --continue save.txt  16jY7qLJnxb7CHZyqBP8qca9d51gAjyXQN -b 54 -t 256 -p 1500 -o keys.txt

the save.txt file creates the application. you shouldn't create it.

---------------------------------
start=0000000000000000000000000000000000000000000000000000000000000001
next=000000000000000000000000000000000000000000000000000000A3EE38B1F4
end=000000000000000000000000000000000000000000000000FFFFFFFFFFFFFFFF
blocks=54
threads=256
points=1500
compression=compressed
device=0
elapsed=1274953
stride=0000000000000000000000000000000000000000000000000000000000000001
------------------------------------

**Comment by ghost** [comment]
SirDennis,
So the problem here is that OP created an empty save.txt?

**Comment by sari777** [comment]
can anyone tell what is the elapsed meaning in resume txt file.? it means the private keys it has searched.? or time.?

**Comment by bill32767** [comment]
Yes, the pk it has searched within a given time period and key space.

**Comment by sari777** [comment]
thanks for answer. and...what it should be if i have 8gb gpu. per minute.?

**Comment by sari777** [comment]
54.15 MKey/s (78,047,608,832 total) what that means.?

**Comment by sari777** [comment]
hey dear. you there.? i have lot of confusion around here
